public/components/trace_analytics/components/common/index.ts

```diff
@@ -0,0 +1,42 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ *
+ * Modifications Copyright OpenSearch Contributors. See
+ * GitHub history for details.
+ */
+
+/*
+ *   Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License").
+ *   You may not use this file except in compliance with the License.
+ *   A copy of the License is located at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   or in the "license" file accompanying this file. This file is distributed
+ *   on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ *   express or implied. See the License for the specific language governing
+ *   permissions and limitations under the License.
+ */
+
+// export * from './helper_functions';
+// export { SearchBar, SearchBarProps, renderDatePicker } from './search_bar';
+// export { Plt } from './plots/plt';
+
+export const RAW_INDEX_NAME = 'otel-v1-apm-span-*';
+export const SERVICE_MAP_INDEX_NAME = 'otel-v1-apm-service-map*';
```

Comment:
Indices and route can be moved to https://github.com/opensearch-project/trace-analytics/tree/feature/observability/common/constants as they are used in the server part too. You may create a new trace-analytics file here.

Reply:
will do, thanks